modules/uid2IdSystem_shared.js

```diff
@@ -165,6 +165,12 @@ function refreshTokenAndStore(baseUrl, token, clientId, storageManager, _logInfo
   });
 }
 
+function isExpired(timestamp) {
+  // Check if the value is in second and convert to milliseconds if necessary
+  let expiredAt = timestamp < 2000000000 ? timestamp * 1000 : timestamp
```

Comment:
2000000000 is Jan 24 1970 - there will definitely be no real tokens with that date, so I guess anything below that is kind of an invalid value. But I'm not sure about treating this as a "maybe in seconds" value - I think I'd prefer to insist it should be in ms (but open to opinions on this!)